Client/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp

```diff
@@ -1076,6 +1076,13 @@ bool CStaticFunctionDefinitions::SetElementPosition(CClientEntity& Entity, const
 {
     RUN_CHILDREN(SetElementPosition(**iter, vecPosition))
 
+    if (Entity.GetType() == CCLIENTVEHICLE)
+    {
+        CClientPed* driver = static_cast<CClientVehicle&>(Entity).GetOccupant(0);
+        if (!Entity.IsLocalEntity() && !static_cast<CDeathmatchVehicle&>(Entity).IsSyncing() && (driver && !driver->IsLocalPlayer()))
```

Comment:
I would forbid setElementPosition when `driver == nullptr `as well. So `(!driver || !driver->IsLocalPlayer())`. What do you think?